### ISSUE TITLE
favicon still using KB 

### ISSUE THREAD

**Issue by jdchandler** [Kikobeats/uno-urban#19]
# General comment

I used http://realfavicongenerator.net as suggested and replaced EVERY picture inside assets/img

After checking everything is using my favicon except Chrome browser, it still shows KB.

The website shows errors, screenshot here - https://i.imgur.com/zUow3Ri.png

The important error is:
http://justinchandler.blog/assets/img/favicon.ico?v=wAAv6Wqe6l and http://justinchandler.blog/favicon.ico both exist and are different

I can't even find "favicon.ico?v=wAAv6Wqe6l" anywhere in the uno-urban folder. Looks like a variable and I don't know whats forcing it. There are no other instructions other then replacing the files.

## Environment Settings
Code Injection:

HEAD:

<script> var disqus_shortname = 'justinchandler'; </script> <script> var posts_headline = 'Thoughts, Stories, Ideas.'; </script>
FOOT:

<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/zepto/1.1.6/zepto.min.js"></script> <script>jQuery = Zepto</script>

Theme version: 3.3.7
Browser: Google Chrome Version 56.0.2924.87 (64-bit)
OS: Windows 10 Anniversary Update

**Comment by jdchandler** [comment]
The favicon website says it can't find these files:

Cannot find http://justinchandler.blog/android-chrome-36x36.png
Cannot find http://justinchandler.blog/android-chrome-48x48.png
Cannot find http://justinchandler.blog/android-chrome-72x72.png
Cannot find http://justinchandler.blog/android-chrome-96x96.png
Cannot find http://justinchandler.blog/android-chrome-144x144.png
Cannot find http://justinchandler.blog/android-chrome-192x192.png
Cannot find http://justinchandler.blog/android-chrome-256x256.png
Cannot find http://justinchandler.blog/android-chrome-384x384.png

But they are in the assets/img directory

chmod 777 *
chown ghost:ghost *

**Comment by Kikobeats** [comment]
`v=wAAv6Wqe6l` is a flag for force load the new favicon icon and prevent load a previously cached icon. 

This is necessary becase this kind of image is cached by the browser frequently.

So, in your case, after `realfavicongenerator.net` you will have a different code, use it.

Also need generate the rest of icon or update the html

**Comment by Kikobeats** [comment]
I enter at http://justinchandler.blog/ and I saw correctly the favicon, so I suppose is resolved.

lol gabe, don't take my money

**Comment by jdchandler** [comment]
If you were on mobile it works there.

I wanna say that the bug is your caching method.

Because on EVERY PC I go to and use CHROME it pulls the KB favicon.

If you didn't read already I replaced EVERY PICTURE inside of asset/img with my own.

So it is like NOT force loading the NEW favicon but rather still reloading the old no longer existing favicon.

Notice the errors from realfavicongenerator.net that I provided.

I used EVERY file they gave me including the new code and replaced EVERY FILE in asset/img

I will admit on MOBILE it uses the correct favicon. Even in Internet Explorer it does. But chrome is stuck with this KB favicon and I can't get rid of it. 

I need to force v=wAAv6Wqe6l to use MY favicon
